sql/hypertable.sql

```diff
@@ -0,0 +1,3 @@
+-- Trigger that blocks INSERTs on the hypertable's root table
+CREATE OR REPLACE FUNCTION _timescaledb_internal.insert_blocker() RETURNS trigger
```

Comment:
Hm, a little confused why this is necessary to expose. Is there no way to keep this entirely internal to the extension? (See below)